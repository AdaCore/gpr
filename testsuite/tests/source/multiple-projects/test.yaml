description:
    Check same sources in multiple projects
driver: build_and_run
project_file: source_multiple_projects
main: main
