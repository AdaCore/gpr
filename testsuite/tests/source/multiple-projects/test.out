Project: Main
demo2.gpr: error: source file "a1.ads" already part of project Demo1
Project: MainC
mainc.gpr: error: source file "a1.ads" already part of project Demo1
Project: MainE
Project: MainA
